hera_pspec/pstokes.py

```diff
@@ -0,0 +1,166 @@
+"""
+Module to construct pseudo-Stokes (I,Q,U,V) visibilities from miriad files or UVData objects
+"""
+import numpy as np, os
+import pyuvdata
+import copy
+
+# weights used in forming Stokes visibilities
+pol_weights = {
+    'pI': {'XX': 1. , 'YY': 1. },
+    'pQ': {'XX': 1. , 'YY':-1. },
+    'pU': {'XY': 1. , 'YX': 1. },
+    'pV': {'XY':-1.j, 'YX': 1.j},
+}
+
+def miriad2pyuvdata(dset):
+   """
+   Converts Miriad file to UVData object
```

Comment:
Might be worth stating explicitly that it opens the file and reads it into a UVData object (as this might be quite a slow/memory-hungry process for some files).